core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -1169,6 +1169,103 @@ class SQLPlanParserSuite extends BasePlanParserSuite {
     }
   }
 
+  test("MonthsBetween is supported in ProjectExec") {
```

Comment:
The following 3 tests are very identical.
Is it possible to create a parameterized test for all of them?